### PR TITLE
Add wait_for_sync_timeout

### PR DESCRIPTION
**Pull request by tsdg112**
Wait_for_sync waited 300 seconds for Mint for sync with the institution.  I added wait_for_sync_timeout to specify the number of seconds to wait.  The default is 300 seconds.

### CODE REVIEW

**Comment by tsdg112** [comment]
A typo in commit e069366a299c7ff22167184728692a8b4a8c644a broke retrieving transactions for a specified account.  The second commit here fixes it.

File: mintapi/api.py [hunk 1]
```diff
@@ -568,7 +572,7 @@ def get_transactions_json(self, include_investment=False,
             # transactions as well.  Otherwise they are skipped by
             # default.
             if id > 0 or include_investment:
-                params['id'] = id
+                params['accountId'] = id
```
Comment (mrooney) [inline mintapi/api.py]: Looks good overall. What is this change for?

**Comment by mrooney** [comment]
Thanks, looks nice, just added one question :)

File: mintapi/api.py (hunk 1 above)
Comment (tsdg112) [reply mintapi/api.py]: This was the typo in commit e069366 that broke retrieving transactions for a specified account. This fixes it.

**Comment by mrooney** [comment]
Thank you for your contribution!

**Pull request merged**